### ISSUE TITLE
Example

### ISSUE THREAD

**Issue by allain** [saiichihashimoto/feathers-react-redux#13]
Hello can you please add a simple example somewhere so we can get a feel for what this might feel like in the wild.

**Comment by laneallen** [comment]
@allain It seems this was just roughed out code ( https://github.com/feathersjs/feathers/issues/264#issuecomment-198804630 ) as a proof of concept rather than an actual working middleware.

**Comment by saiichihashimoto** [comment]
Yes, this is true. This isn't fully developed and I don't have any plans to pursue it.
